release_tester/arangodb/installers/__init__.py

```diff
@@ -79,5 +80,8 @@ def make_installer(install_config: InstallerConfig):
         if distro[0] in ['centos', 'redhat', 'suse']:
             from arangodb.installers.rpm import InstallerRPM
             return InstallerRPM(install_config)
+        if distro[0] in ['debian', 'ubuntu']:
```

Comment:
this needs to be `if install_config.zip:` and needs to be in front of line 77. 
the `if` statements above need to be `elsif`. 
Add this `if` statement also to line 69 - `if macver[0]:`